helm-chart/renku-notebooks/values.yaml

```diff
@@ -98,9 +98,8 @@ jupyterhub:
       tag: 'latest'
     allowNamedServers: true
     services:
-      gateway:
-        oauth_no_confirm: true
       notebooks:
+        oauth_no_confirm: true
```

Comment:
was this just in the wrong place before?

Reply:
No. For the gateway I have moved this information to the renku chart because there is no gateway service registered with the Jupyterhub when running as a secondary deployment for anonymous sessions (see our long discussion on the Renku chart ;). 
Why the missing `oauth_no_confirm: true` on the notebooks service didn't result in any confirmation screen showing up in our "regular" flow I don't know, but when using the tmpauthenticator it is needed.